app/views/shared/metadata/edit/_study.html.erb

```diff
@@ -16,9 +16,10 @@
       <%= group.text_area(:study_description) %>
       <%= group.text_area(:study_abstract) %>
       <%= group.radio_select(:study_sra_hold, Study::STUDY_SRA_HOLDS) %>
-      <%= metadata_fields.select(:ebi_library_strategy, Study::EBI_LIBRARY_STRATEGY_OPTIONS) %> 
-      <%= metadata_fields.select(:ebi_library_source, Study::EBI_LIBRARY_SOURCE_OPTIONS) %> 
-      <%= metadata_fields.select(:ebi_library_selection, Study::EBI_LIBRARY_SELECTION_OPTIONS) %> 
+     
+      <%= metadata_fields.select(:ebi_library_strategy, options_for_select([['Please select...', nil]] + Study::EBI_LIBRARY_STRATEGY_OPTIONS, metadata_fields.object.ebi_library_strategy), (metadata_fields.object.new_record? ? { required: true } : {})) %>
+      <%= metadata_fields.select(:ebi_library_source,  options_for_select([['Please select...', nil]] + Study::EBI_LIBRARY_SOURCE_OPTIONS, metadata_fields.object.ebi_library_source), (metadata_fields.object.new_record? ? { required: true } : {})) %> 
+      <%= metadata_fields.select(:ebi_library_selection, options_for_select([['Please select...', nil]] + Study::EBI_LIBRARY_SELECTION_OPTIONS, metadata_fields.object.ebi_library_selection), (metadata_fields.object.new_record? ? { required: true } : {})) %> 
```

Comment:
What is the purpose of the nil option? The field is required, `*_OPTIONS` are populated from the ena_requirements_fields.yml and there is a 'Please select...'  placeholder option.